src/get-started/webpack.md

```diff
@@ -1,8 +1,45 @@
 ---
 layout: page
-title: Get Started > Webpack
+title: Get Started > webpack
 ---
 
-# Webpack
+# webpack
 
-Need to write up.
+The following example uses [workbox-webpack-plugin](https://www.npmjs.com/package/workbox-webpack-plugin)
+to create a precaching service worker in your webpack build process. This will
```

Comment:
👍 Matt's phrasing "The following examples uses the workbox-webpack-plugin to generate a service worker file capable of precaching your site for offline use."